src/index.html

```diff
@@ -9,7 +9,7 @@
 <body>
   <nav>
     <div>
-      <h2>FitLit</h2>
+      <h1>FitLit</h1>
```

Comment:
Do we still have h2s anywhere?  Another accessibility thing with screen readers is that we don't skip any heading numbers (e.g. jumping from h1 to h3), so just making sure we still have the h2s somewhere.

Reply:
There is markdown interpolated in the index.js file. That is where the h2 is. I agree, the flow of the headings is not accessible. @trishalanglois do want to change this?